frontend/online-shop/src/styles/productDetails/ProductDetailsStyle.tsx

```diff
@@ -0,0 +1,33 @@
+import { createStyles, makeStyles, Theme } from "@material-ui/core";
+
+export const useStyle = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      overflow: "hidden",
+    },
+    info: {
+      width: "100%",
+    },
+    img: {
+      objectFit: "cover",
+      maxWidth: "50%",
+      maxHeight: "100vh",
+    },
+    grid: {
+      backgroundColor: theme.palette.primary.dark,
+    },
+    div: {
+      color: "white",
+      fontSize: "25px",
+      margin: "0 auto",
+      textAlign: "center",
+      lineHeight: "16.666vh",
```

Comment:
I honestly don't know what this is helping with but I would recommend to find a different solution. It could really affect the styling.